operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md

```diff
@@ -329,9 +367,9 @@ There should be a cray-cps pod (the broker), three cray-cps-etcd pods and their
     [Tue Jul 21 13:09:54 2020] DVS: merge_one#358:   Ignoring.
     ```
 
-19. Make sure the Configuration Framework Service (CFS) finished successfully. Review *HPE Cray EX DVS Administration Guide 1.4.1 S-8004*.
+20. Make sure the Configuration Framework Service (CFS) finished successfully. Review *HPE Cray EX DVS Administration Guide 1.4.1 S-8004*.
```

Comment:
- Should we provide a command here to ensure CFS finished successfully? Not sure if that would be with `cray cfs components describe` or if another command is needed.
- What is the purpose of reviewing the DVS admin guide here? Where should the user be looking in the guide if the link is still needed?

Reply:
I'm not sure, as that is what is currently present in https://github.com/Cray-HPE/docs-csm/blob/main/operations/node_management/Swap_a_Compute_Blade_with_a_Different_System.md?plain=1#L611 which is where this procedure was derived from.